lambda/lambda_loader.py

```diff
@@ -44,17 +46,10 @@ def handler(event, context):
     log.debug("got lambda path")
     json_event = json.dumps(event)
     print("event: " + json_event)
-    args = ("bin/python3.4", lambda_path, json_event)
-
-    # AWS defines a PYTHONPATH that breaks our Python 3.4 code.  We can't
-    # import concurrent.futures, specifically.
-    env = os.environ
-    if "PYTHONPATH" in env:
-        del env['PYTHONPATH']
-
-    popen = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=env)
-    popen.wait()
-    output = popen.stdout.read()
-    err_str = popen.stderr.read()
-    print("output: " + output)
-    print("err: " + err_str)
+
+    # Pass to lambda function we're about to load.
+    if len(sys.argv) > 1:
```

Comment:
What prompted this / why have a conditional here? It looks like it would place json_event in a different index in the array.

Reply:
All our current lambda functions expect the json arguments as the first argument. I figure that this is just a stop-gap before we actually break up the multi-lambda into separate lambdas. The conditional is just in case there's an argument passed to the lambda loader (there's not, currently).